projects/project-player-library/src/lib/shared/attachment-show-card/attachment-show-card.component.html

```diff
@@ -1,8 +1,10 @@
 @if (attachments.type.includes('image')) {
     <mat-card class="w-full  rounded-full p-2 " style=" box-shadow: rgba(133, 119, 119, 0.555) 5px 5px 4px;">
-        <div class="flex w-full justify-end">
-            <mat-icon  class="primary-icon cursor-pointer" fontIcon="cancel" (click)="actionEmit(attachments)"></mat-icon>
-        </div>
+        @if ( projectDataStatus !== 'submitted' ) {
```

Comment:
Use the 'submitted' value from the constant file

Reply:
done